docs/features/resource-reservation.md

```diff
@@ -0,0 +1,234 @@
+# Resource Reservation User Documentation
+
+This document uses a kind cluster as an example to introduce how to enable and use the resource reservation capability of the Godel Scheduler.
+
+## Prerequisites
+
```

Comment:
But how to deploy godel controller manager, is there any documentation, or links maybe?

Reply:
Follow the `docs/features/kind-cluster-setup.md`, users can deploy godel controller manager in kind cluster.